ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -468,18 +464,15 @@ static KsqlRestApplication buildApplication(
         restConfig,
         versionCheckerFactory,
         Integer.MAX_VALUE,
-        serviceContext,
-        KsqlSecurityContextBinder::new);
+        serviceContext);
   }
 
   static KsqlRestApplication buildApplication(
       final String metricsPrefix,
       final KsqlRestConfig restConfig,
       final Function<Supplier<Boolean>, VersionCheckerAgent> versionCheckerFactory,
       final int maxStatementRetries,
-      final ServiceContext serviceContext,
-      final BiFunction<KsqlConfig, KsqlSecurityExtension, Binder> serviceContextBinderFactory
```

Comment:
Is this safe to remove? I think it was added to mock some unit tests. If they're passing, then I'm good with this, but if not, then you might need to return it.

Reply:
Yes, I discovered this was required.  Added it back.